src/drivers/imx/interrupt.c

```diff
@@ -347,9 +347,8 @@ static inline void irq_handler(void *data, uint32_t line_index)
 
 		if (!--tries) {
 			tries = IRQ_MAX_TRIES;
-			tr_err(&irq_i_tr, "irq_handler(): IRQ storm, status "
-			       PRIx64,
-			       get_irqsteer_interrupts(line_index));
+			tr_err(&irq_i_tr, "irq_handler(): IRQ storm, status 0x%08x%08x",
+			       (uint32_t)(status >> 32), (uint32_t)status);
```

Comment:
Almost perfect, except the correct print is the one with the two halves switched around (I want to see higher order bits first). Thanks.

Reply:
@paulstelian97 I think it's correct, in fact even tested. It's arithmetic shifting and casting, if I accessed parts using a pointer like `((unsigned int *)&int64)[0]` then I'd need to take endianness into account and swap order respectively

Reply:
Yeah nevermind. My bad.